pkg/rancher-components/public/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
```

Comment:
What's the plan for these public assets? There's a lot of work going on for with storybook, Neil has also mentioned we should think about maybe a public site. My fear is that we would start duplicating effort if this became another visual guide to dashboard assets

Reply:
We would need for sure 2 Storybooks: one as living styleguide for the component library, one for the UI of the dashboard, so we could verify the abstracted UI.
I am not so sure how much can be useful the second one, though.

Reply:
I think this is a vestige of boilerplate and can be safely removed.. I would prefer that we don't duplicate these efforts as well 🙂